index/completely-indexed.yml

```diff
@@ -0,0 +1,21 @@
+# WRK:
+#   complete: False
+#   index_author:
+#   - family: Doe
+#     given: John
+#   - family: Doe
+#     given: Jane
+#   sections_complete:
+#   - end: 32
+#     start: 25
+#   - start: 20
+
+G:
+  complete: True
+  index_author:
+  - family: Delft
+    given : André
+    dropping-particle: van
+  - family: Wit
+    given: Johannes
+    dropping-particle: de
```

Comment:
Het voelt niet helemaal lekker om voor zo'n 'kleine' feature een aparte file te gebruiken. Volgens mij kun je deze velden best toevoegen aan objecten van `works.yaml`, de citeproc server negeert volgens mij gewoon de velden die niet bij de CSL spec horen.

Dan zou overigens `complete` misschien wel een wat onduidelijke naam zijn. `completely_indexed` wellicht dan gewoon. (Of `index_status`, voor als we in het vervolg nog andere statussen willen toevoegen..)

Reply:
Ik zal ze gewoon in works.yml zetten. Ik had ze in een aparte file gezet omdat het niet CSL was, maar als de server ze gewoon negeerd kan dat inderdaad prima!